base/bucket_test.go

```diff
@@ -231,9 +230,59 @@ func TestGetStatsVbSeqno(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(ts *testing.T) {
 			actualUUIDs, actualHighSeqnos, err := GetStatsVbSeqno(test.stats, maxVbno, false)
-			goassert.Equals(ts, err, nil)
-			goassert.DeepEquals(ts, actualUUIDs, test.expectedUUIDs)
-			goassert.DeepEquals(ts, actualHighSeqnos, test.expectedHighSeqnos)
+			assert.NoError(ts, err)
+			assert.Equal(t, test.expectedUUIDs, actualUUIDs)
+			assert.Equal(t, test.expectedHighSeqnos, actualHighSeqnos)
 		})
 	}
 }
+
+func TestChooseCouchbaseDriver(t *testing.T) {
+	assert.Equal(t, GoCBCustomSGTranscoder, ChooseCouchbaseDriver(DataBucket))
+	assert.Equal(t, GoCB, ChooseCouchbaseDriver(IndexBucket))
+	unknownCouchbaseBucketType := CouchbaseBucketType(math.MaxInt8)
+	assert.Equal(t, GoCB, ChooseCouchbaseDriver(unknownCouchbaseBucketType))
+}
+
+func TestCouchbaseDriverToString(t *testing.T) {
+	assert.Equal(t, "GoCB", GoCB.String())
+	assert.Equal(t, "GoCBCustomSGTranscoder", GoCBCustomSGTranscoder.String())
+	unknownCouchbaseDriver := CouchbaseDriver(math.MaxInt8)
+	assert.Equal(t, "UnknownCouchbaseDriver", unknownCouchbaseDriver.String())
+}
+
+func TestIsTLS(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	fakeBucketSpec.Server = "http://localhost:8091"
+	assert.False(t, fakeBucketSpec.IsTLS())
+	fakeBucketSpec.Server = "https://localhost:443"
+	assert.True(t, fakeBucketSpec.IsTLS())
+	fakeBucketSpec.Server = "couchbases"
+	assert.True(t, fakeBucketSpec.IsTLS())
+}
+
+func TestUseClientCert(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	assert.False(t, fakeBucketSpec.UseClientCert())
+	fakeBucketSpec.Certpath = "/var/lib/couchbase/inbox/ca.pem"
+	assert.False(t, fakeBucketSpec.UseClientCert())
+	fakeBucketSpec.Keypath = "/var/lib/couchbase/inbox/pkey.key"
+	assert.True(t, fakeBucketSpec.UseClientCert())
+}
+
+func TestGetPoolName(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	assert.Equal(t, "default", fakeBucketSpec.GetPoolName())
+	fakeBucketSpec.PoolName = "Liverpool"
+	assert.Equal(t, "Liverpool", fakeBucketSpec.GetPoolName())
+}
+
+func TestGetViewQueryTimeout(t *testing.T) {
+	fakeBucketSpec := &BucketSpec{}
+	expectedViewQueryTimeout := time.Duration(75*1000) * time.Millisecond
+	assert.Equal(t, expectedViewQueryTimeout, fakeBucketSpec.GetViewQueryTimeout())
```

Comment:
I agree with not duplicating the calculation in the function, so either comparing against `75000` directly, or if using durations, `75 * time.Second`?

My point was it's easier to see what `75000` actually is vs. `0x124f8` when reading a test.

Reply:
Makes more sense. I've committed the changes against the latest revision.